skills/ios-simulator-testing/scripts/app_state_capture.py

```
@@ -0,0 +1,292 @@
+        """Capture accessibility tree using shared utility."""
+        try:
+            # Use shared utility to fetch tree
+            tree = get_accessibility_tree(self.udid, nested=True)
+
+            # Save tree
+            with open(output_path, "w") as f:
+                json.dump(tree, f, indent=2)
+
+            # Return summary using shared utility
+            return {"captured": True, "element_count": count_elements(tree)}
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)